build.gradle

```diff
@@ -319,6 +319,7 @@ dependencies {
   compile("io.prometheus:simpleclient_servlet:${prometheusVersion}")
   compile("io.prometheus:simpleclient_spring_boot:${prometheusVersion}")
 
+  compile("com.github.jasminb:jsonapi-converter:${jsonapiConverterVersion}")
```

Comment:
I don't like to ship an end user library in the server. We are adding annotations into our domain classes just for the sake of integration testing. That adds unnecessary complexity, along with confusing overloading of @Id annotation.
We should go for static json strings instead.

@micheljung Your opinion on this?

Reply:
Ok result from the discord discussion:
We want to use a separate repository (i.e. faf-java-commons) for the client-side dtos which have the jsonapi-converter annotations. These can be included with testcompile and used in the integration tests. These DTOs are needed in java-client and moderator-client anyway so testing them in the api gives additional test coverage.